modules/reissue-internal-certificates-central.adoc

```diff
@@ -7,21 +7,31 @@
 
 Central uses a built-in server certificate for authentication when communicating with other {product-title} services.
 This certificate is unique to your Central installation.
-The RHACS portal shows an information banner when the Central certificate is about to expire.
+The {product-title-short} portal shows an information banner when the Central certificate is about to expire.
 
 [NOTE]
 ====
-The information banner only appears 15 days before the certificate expiry date.
+The information banner only appears 15 days before the certificate expiration date.
 ====
 
+For Operator-based installations, beginning with {product-title-short} version 4.3.4, the Operator will automatically rotate all Central components' service TLS certificates 6 months before they expire. The following conditions apply:
```

Comment:
is central capitalized? I see it lowercase [here](https://docs.openshift.com/container-platform/4.15/operators/olm_v1/arch/olmv1-components.html) maybe its not the same thing, but I just thought I'd ask.

Reply:
I see how it is used on down so maybe not??

Reply:
Yeah, for ACS it is a component, so it is capitalized in the doc set.